### DISCUSSION TITLE
Request: New transition

### DISCUSSION THREAD

**Discussion by fe2o3fish** [simpleavr/s14clock#21] [Ideas]
Slide from left to right, replacing characters one-by-one.

BTW, if it wasn't obvious, I'm really enjoying my clock!!

**Comment by simpleavr** [comment]
Actually I tried this one before (not left to right though, it was like shift, right to left). It wasn't appealing to me and it takes longer time to cycle one message.

I might add it back in the next **beta firmware** for user comments.

**Comment by simpleavr** [comment]
**Add direction override (right to left) for Shift and Drip transitions**
add `~=` in your content entry for shifting / dripping message from right to left, instead of left to right.

Again, please go to the [download page](https://simpleavr.github.io/s14clock/install.html) and choose the beta firmware to download.
Will work on better descriptions and additional explanations in project page later.
